pages/_app.js

```diff
@@ -0,0 +1,41 @@
+/* global process */
+
+import React from 'react'
+import App from 'next/app'
+import * as Sentry from '@sentry/browser'
+
+Sentry.init({
+  dsn: process.env.SENTRY_DSN
+})
+
+class MyApp extends App {
```

Comment:
Where is this app used?

Reply:
It's replacement of the standard Next.js component https://nextjs.org/docs/advanced-features/custom-app . Basically it's a wrapper around the pages.